src/sentiment.jl

```diff
@@ -35,7 +35,7 @@ function flatten(x)
     return reshape(x, (l, 1))
 end
 
-function get_sentiment(ip::Array{T, 1}, weight, rwi) where T <: AbstractString
+function get_sentiment(ip::Array{T, 1}, weight, rwi, handle_unknown) where T <: AbstractString
```

Comment:
```suggestion
function get_sentiment(ip::AbstractVector{<:AbstractString}, weight, rwi, handle_unknown)
```

also `handle_unknown` as a function argument should be first.
https://docs.julialang.org/en/v1/manual/style-guide/index.html#Write-functions-with-argument-ordering-similar-to-Julia-Base-1